src/Rules/StarterWeb/IndividualAuth/Views/Account/Login.cshtml

```diff
@@ -33,8 +33,11 @@
                 <div class="form-group">
                     <div class="col-md-offset-2 col-md-10">
                         <div class="checkbox">
-                            <input asp-for="RememberMe" />
-                            <label asp-for="RememberMe"></label>
+                            @* fix https://github.com/aspnet/Mvc/issues/2369 *@
```

Comment:
@NTaylorMullen Thanks! @peterblazejewicz can you please update this PR as per @NTaylorMullen comments?